modules/game/src/main/GameRepo.scala

```diff
@@ -159,6 +160,56 @@ final class GameRepo(val coll: Coll)(implicit ec: scala.concurrent.ExecutionCont
   ): AkkaStreamCursor[Game] =
     coll.find(selector).sort(sort).batchSize(batchSize).cursor[Game](readPreference)
 
+  def getCorrespondenceGamesEmailNotif(userColl: Coll, prefColl: Coll) =
```

Comment:
The query. Probably need further optimisation. Also not yet a streaming query as I wasn't sure how to handle it.